src/pages/InfoList/InfoFeed.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const InfoFeed = () => {
+  return (
+    <InfoFeedContainer>
+      <div>InfoFeed 입니다.</div>
+    </InfoFeedContainer>
+  );
+};
+
+const InfoFeedContainer = styled.div`
```

Comment:
```suggestion
const Container = styled.div`
```
컴포넌트들 네이밍 스코프가 겹치지 않기 때문에 저라면 간단하게 가장 바깥쪽 요소는 컨테이너라고 지을거 같아요. 이렇게 바꿀 수 있는 요소들이 많이 있는거 같네요.

Reply:
넵 확인하고 해당 요소들 네이밍 다 수정하도록 하겠습니다!